cmd/lift/lift.go

```diff
@@ -27,6 +39,24 @@ func lift(chainFile string, inFile string, outFile string) {
 			log.Fatalf("Multiple overlaps???")
 		}
 		i.UpdateLift(interval.LiftIntervalWithChain(overlap[0].(*chain.Chain), i))
+		//special check for lifting over VCF files
+		if faFile != "" {
+			//faFile will be given if we are lifting over VCF data.
```

Comment:
Do we need a check for this to ensure we get an error if an fa is given with a non-vcf, or if a vcf is given without an fa?